bin/rails

```diff
@@ -1,14 +1,6 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-# for separate processes, for example, via selenium
-# see https://github.com/colszowka/simplecov#getting-started
-if ENV['RAILS_ENV'] == 'test'
-  require 'simplecov'
-  SimpleCov.start 'rails'
-  puts "required simplecov"
-end
```

Comment:
Answer from @pking-moj: 

> Ruby 3.1 throws an error if we call start twice. We call it for the RSpec helper and the rails was a "catch all" so that seems the logical place to remove it. Code Climate it still working as expected so reports are still being produced as we would expect